animal/noise.py

```diff
@@ -0,0 +1,52 @@
+"""This module contains Environment wrappers for noise observation and action noise."""
+
+import numpy as np
+from gym import Env
+from gym.core import ActionWrapper, ObservationWrapper
+from gym.spaces import Box, Discrete
+
+
```

Comment:
Wrappers mostly look good, but:

1. I think for the observation one it's necessary to add the `step` and `reset` functions so that noise is applied to the observations before they are given to the agent. 
2. Also, for the action one I think we need a `step` function but probably not a `reset` function

Reply:
Yep. Mentioned on Discord, but [ActionWrapper](https://github.com/openai/gym/blob/master/gym/core.py#L320-L334) and [ObservationWrapper](https://github.com/openai/gym/blob/master/gym/core.py#L320-L334) have these built in.